tekton/ui/pipeline.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ui-pipeline
+spec:
+  params:
+    - name: HUB_REPO
+    - name: REVISION
+    - name: IMAGE
+    - name: TAG
+    - name: HUB_NAMESPACE
+    - name: K8S_VARIANT
+  workspaces:
+    - name: shared-workspace
+    - name: kubeconfig-dir
+      optional: true
+  tasks:
+    - name: fetch-repository
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: url
+          value: $(params.HUB_REPO)
+        - name: subdirectory
+          value: ""
+        - name: deleteExisting
+          value: "true"
+        - name: revision
+          value: $(params.REVISION)
+    - name: install-dependencies
+      taskRef:
+        name: npm
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+      params:
+        - name: PATH_CONTEXT
+          value: "ui"
+        - name: ARGS
+          value:
+            - clean-install
+      runAfter:
+        - fetch-repository
+    # - name: run-lint
+    #   taskRef:
+    #     name: npm
+    #   workspaces:
+    #     - name: source
+    #       workspace: shared-workspace
+    #   params:
+    #     - name: PATH_CONTEXT
+    #       value: "ui"
+    #     - name: ARGS
+    #       value:
+    #         - run
+    #         - lint
+    #   runAfter:
+    #     - install-dependencies
```

Comment:
I guess the lint check is now green so should we enable this?
If not then can you please remove this?